bot/modules/bartek.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import discord
+from discord.ext import commands
+
+from ..bot import DegeneratBot
+
+import csv
+import os
+from datetime import date
+
+PATH = "./data/polityka"
+
+class Bartek(commands.Cog):
+    def __init__(self, bot: DegeneratBot):
+        self.bot: DegeneratBot = bot
+
+        guild_id = os.getenv("GUILD_ID")
+        if not guild_id:
+            return logging.fatal("GUILD_ID environment variable not set!")
+
+        bartek_id = os.getenv("BARTEK_ID")
+        if not bartek_id:
+            return logging.fatal("BARTEK_ID environment variable not set!")
```

Comment:
This could be moved to the module setup function at the very bottom of the file. It would skip adding the cog all together if the variables are not set.